link/src/main/java/com/stripe/android/link/ui/verification/VerificationScreen.kt

```diff
@@ -161,7 +162,8 @@ internal fun VerificationBody(
 }
 
 @Composable
-internal fun VerificationBody(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun VerificationBody(
```

Comment:
Why is this necessary?

Reply:
oops reverted - https://github.com/stripe/stripe-android/pull/6005/commits/4a07afec616023a46bcae91e8b7b3419f8c316e6